check/core/string_check.c

```diff
@@ -241,7 +241,7 @@ bool_t check_string_merge(void) {
 			total += *(st_char_get(strings[15]) + i);
 		}
 
-		if (total != (5376lu * 16lu)) {
+		if (total != 5376UL * 16)) {
```

Comment:
You probably didn't mean to delete that left paren...

Try this instead: `if (total != (5376UL * 16)) {`